### DISCUSSION TITLE
Build without evolution data server

### DISCUSSION THREAD

**Discussion by jonas-w** [andyholmes/valent#371] [Feature Request]
Hey,

First, thank you very much for building a kde connect in GTK, i can't really stand qt and always missed kde connect, because nothing is as good.

Aas i don't want/need the contacts plugin, i would like to build valent without the evolution dataserver installed.

Would this be possible?

I don't have much clue about C or Meson and tried to disable them in meson_options.txt, but building still fails without the evolution data server dependency installed.

**Comment by andyholmes** [comment]
Sorry, Valent has a hard dependency on `libevolution-data-server`. It doesn't need the service to be running, but it does need the library.

**Reply by andyholmes** [reply]
FWIW I may revisit this "sometime soon". The only public symbol I'm using is `EContact`, so I may be able to push all the Evolution dependencies into the `eds` plugin, in which case that plugin can be disabled making the dependency optional.
